mapzen-places-api/src/main/java/com/mapzen/places/api/Place.java

```diff
@@ -1,7 +1,203 @@
 package com.mapzen.places.api;
 
+import android.net.Uri;
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.text.TextUtils;
+
+import java.util.List;
+import java.util.Locale;
+
 /**
  * Represents a physical place selected from a {@link com.mapzen.android.graphics.MapzenMap}.
  */
-public class Place {
+public class Place implements Parcelable {
```

Comment:
What do you think about converting `Place` to a public interface and putting the constructor in an internal implementation file?